Lib/idlelib/calltips.py

```diff
@@ -123,6 +123,8 @@ def get_entity(expression):
 _INDENT = ' '*4  # for wrapped signatures
 _first_param = re.compile(r'(?<=\()\w*\,?\s*')
 _default_callable_argspec = "See source or doc"
+_invalid_method = "invalid method signature"
+_argument_positional = "('/' marks preceding arguments as positional-only)"
```

Comment:
By 'word', I presume you meant 'positional-only'.  Nice check. We'll go with your preference.